api/config/config_prod.json

```diff
@@ -30,7 +30,10 @@
       0.5,
       0.75
     ],
-    "carahsoftEmail": "NIHStrides@Carahsoft.com"
+    "carahsoftEmail": "NIHStrides@Carahsoft.com",
+    "freeTierCronUserBatchSize": 100,
+    "minutesBeforeLastFreeTierJob": 60,
+    "numberOfDaysToConsiderForFreeTierUsageUpdate": 7
```

Comment:
I chose this value to be 7 in prod to account for delayed charges occurring in BQ up to 7 days after use. I am not sure how long it might take for the charge to reflect so I thought 1 week should be enough.